src/libraries/actionPrimitives/CMakeLists.txt

```diff
@@ -8,15 +8,27 @@ set(folder_source src/actionPrimitives.cpp)
 set(folder_header include/iCub/action/actionPrimitives.h)
 
 add_library(${PROJECT_NAME} ${folder_source} ${folder_header})
+add_library(ICUB::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
```

Comment:
I might be new to this but what is it for?

Reply:
I meant why we need it...

Reply:
@traversaro correct me if I am wrong, but it has not a real functionality, just aestetic since targets both with and without namespace are exported.
I personnally prefere the one with the namespace, because it is clearer that it is a COMPONENT of ICUB package